src/services/formatting/rulesMap.ts

```diff
@@ -1,60 +1,59 @@
-///<reference path='references.ts' />
+/// <reference path="rules.ts" />

```

Comment:
do not think we need any of these `/// <references` thourhg out the code base now that we have tsconfig.json files everywhere.. but i guess that is another change.